decimal.go

```diff
@@ -928,8 +928,8 @@ func unquoteIfQuoted(value interface{}) (string, error) {
 	return string(bytes), nil
 }
 
-// NullDecimal represents a fixed-point decimal. It is immutable.
-// number = value * 10 ^ exp
+// NullDecimal represents a nullable decimal with compatibility for
+// scaning null values from the database.
```

Comment:
Minor spelling error, "scaning" should be "scanning" -- otherwise this comment update is much needed 👍

Reply:
Oops, I'll fix that